UI/css/style.css

```diff
@@ -255,6 +285,42 @@ border-radius: 2px;
     font-weight: 600;
     text-shadow: 0 2px 5px rgba(0, 0, 0, 0.5);
 }
+#map {
+    width: 95%;
+    margin: 16px 5px;
+    height: 300px;
+    background-color: grey;
+}#map {
```

Comment:
Why is the #map block repeated

Reply:
it was a typo when pasting accidentaly pasted it twice